java/arcs/core/data/ParticleSpec.kt

```diff
@@ -11,17 +11,16 @@
 
 package arcs.core.data
 
-/**
- * This class contains metadata about a [Particle] in a [Recipe].
- *
- * @property name the name of the particle.
- * @property connections all the handle connections of the particle indexed by the connection name.
- * @property location the location of the implementation.
- */
+/** This class contains metadata about a [Particle] in a [Recipe]. */
```

Comment:
This comment is wrong FWIW.

This class contains metadata about a [Particle] irrespective of a recipe.

So maybe:

/** Specification of an Arcs [Particle]. */ ?

Reply:
Done